site/content/docs/4.3/examples/floating-labels/floating-labels.css

```diff
@@ -48,10 +48,6 @@ body {
   color: transparent;
 }
 
-.form-label-group input:-ms-input-placeholder {
```

Comment:
This doesn't seem like it should be here.

Reply:
Well, it's currently on `master`: https://github.com/twbs/bootstrap/blob/master/site/content/docs/4.3/examples/floating-labels/floating-labels.css#L55

Did I miss something?